common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -20,6 +25,8 @@ public class OverlayManager {
 
     private static final Set<Overlay> enabledOverlays = new HashSet<>();
 
+    private static final List<Pair<Coordinate, Coordinate>> sections = new ArrayList<>(9);
```

Comment:
`Pair<Coordinate, Coordinate>` could be made more descriptive.

Maybe something like

```java
public record BoxCoords(int x1, int x2, int y1, int y2) {}
```